tbtc-ts/src/bitcoin.ts

```diff
@@ -83,6 +90,73 @@ export type UnspentTransactionOutput = TransactionOutpoint & {
   value: number
 }
 
+/**
+ * Represents data of a decomposed transaction.
+ */
+export interface DecomposedRawTransaction {
+  /**
+   * Transaction's version.
+   */
+  version: string
+
+  /**
+   * All transaction's inputs, prepended by the number of transaction inputs.
+   */
+  inputs: string
+
+  /**
+   * All transaction's outputs prepended by the number of transaction outputs.
+   */
+  outputs: string
+
+  /**
+   * Transaction's locktime.
+   */
+  locktime: string
+}
+
+/**
+ * Proof that a given transaction is included in the Bitcoin blockchain.
```

Comment:
`Data required to perform a proof that a given transaction was included in the Bitcoin blockchain.`?

Reply:
Done.